plasmoid/contents/ui/ConfigGeneral.qml

```diff
@@ -17,22 +17,76 @@ Item {
     width: childrenRect.width
     height: childrenRect.height
 
+    property alias cfg_locationEntry: locationEntry.checked
+    property alias cfg_woeidEntry: woeidEntry.checked
     property alias cfg_woeid: woeidField.text
     property alias cfg_interval: intervalField.text
     property alias cfg_timeFormat24: timeFormat24Field.checked
     property alias cfg_useWxFonts: useWxFontsField.checked
+    property bool locChecked: plasmoid.configuration.locationEntry
```

Comment:
locChecked and woeChecked keep the state of the radio buttons for use in the "on clicked" functions. They seem to be needed since once you are inside the on-clicked functions the plasmoid.configuraation.locationEntry/woeidEntry values match the button state clicked on. These addiitional variables hold the "current" state right before the button is clicked.
So I think this are needed. (Actually, one variable would be enough since these are always the binary inverse of each other. However, for clearer logic, I think having two variables is better.)